common/src/models/request/twoFactorEmailRequest.ts

```diff
@@ -2,10 +2,4 @@ import { PasswordVerificationRequest } from './passwordVerificationRequest';
 
 export class TwoFactorEmailRequest extends PasswordVerificationRequest {
     email: string;
-
-    constructor(email: string, masterPasswordHash: string) {
-        super();
-        this.masterPasswordHash = masterPasswordHash;
-        this.email = email;
-    }
```

Comment:
We hate constructors now or something?

Reply:
It's to make it compatible with the factory pattern in `TwoFactorBaseComponent.buildRequestModel` (which in turn uses `UserVerificationService.buildRequest`). That factory assumes that the model you want to build does not take any args in the ctor, which was true for every other request model except for this one, so this seemed like the easiest fix.

Reply:
Thanks for the explanation, Thomas.